xml/System.Text.Json.Serialization/JsonPolymorphicTypeConfiguration`1.xml

```diff
@@ -67,9 +67,9 @@
       </TypeParameters>
       <Parameters />
       <Docs>
-        <typeparam name="TDerivedType">To be added.</typeparam>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <typeparam name="TDerivedType">The derived type with which to associate a type identifier.</typeparam>
+        <summary>Associates specified derived type with supplied string identifier.</summary>
```

Comment:
```suggestion
        <summary>Associates the specified derived type with a supplied string identifier.</summary>
```

Reply:
Q: How is the string identifier supplied?

Reply:
@eiriktsarpalis @layomia can you chime in here?

Reply:
@carlossanlop It looks like this hasn't been resolved - probably the summary needs to be changed?

Reply:
I changed all 3 summaries to `a supplied` instead of `the specified` to preserve the original message, although I also have the same question you asked. It's not clear how that string is supplied. So I think we can address it in a separate issue.